src/components/forms/SubnetTierForm.js

```diff
@@ -244,6 +244,7 @@ class SubnetTierForm extends React.Component {
                     ? "Advanced configuration cannot be set when using dynamically scaled subnets"
                     : "Enable advanced subnet configuration such as custom CIDR blocks",
                   alignModal: "bottom",
+                  align: "left"
```

Comment:
Was this causing some overlap somewhere?

Reply:
Yeah, this was displaying the tooltip above the icon and getting cut off by the top edge of the modal.

Reply:
IDK why it was displaying above despite the alignModal saying "bottom" but it was